xmppserver/src/main/webapp/session-details.jsp

```diff
@@ -356,11 +357,50 @@
             Invalid session/connection
                 <% }
             } %>
+        </td>
     </tr>
 </tbody>
 </table>
 </div>
 
+<br>
+<%  // Show Software Version if there is :
+    if (currentSess.getSoftwareVersion().size() > 1) {
+%>
+    <div class="jive-table">
+        <table cellpadding="3" cellspacing="1" border="0" width="100%">
+            <thead>
+                <tr>
+                    <th colspan="2">
+                        <fmt:message key="session.details.software_version"/>
+                    </th>
+                </tr>
+            </thead>
+            <tbody>
+                <% 
+                try { 
+                        Map<String, String> treeMap = new TreeMap<String, String>(currentSess.getSoftwareVersion());
+                        for (Map.Entry<String, String> entry : treeMap.entrySet()){ %>
+                            <tr>
+                                <td class="c1">
+                                    <%= StringUtils.escapeHTMLTags(entry.getKey().substring(0, 1).toUpperCase()+""+entry.getKey().substring(1)) %>:
+                                </td>
+                                <td>
+                                    <%= StringUtils.escapeHTMLTags(entry.getValue())%>
+                                </td>
+                            </tr>
+                        <% 
+                        }
+                } catch (java.net.UnknownHostException e) { %>
```

Comment:
I don't think that `UnknownHostException` can be thrown in this block, can it? It should be safe to remove the `try/catch` here